app/forms/concept_form.rb

```diff
@@ -0,0 +1,13 @@
+class ConceptForm
```

Comment:
@lsitu Just curious there is model class app/models/concept.rb available which can define fields. Why add separate app/models/forms/concept_form.rb?  Is that necessary?
Otherwise it looks fine to me.